sdk/python/tests/dsl/pipeline_volume_tests.py

```diff
@@ -70,3 +71,8 @@ def test_omitting_name(self):
         name2 = "provided"
         self.assertEqual(vol1.name, name1)
         self.assertEqual(vol2.name, name2)
+
+        # Testing json.dumps() when pvc is a PipelineParam to avoid
+        # `TypeError: Object of type PipelineParam is not JSON serializable`
+        param = PipelineParam(name="foo")
+        vol3 = PipelineVolume(pvc=param)
```

Comment:
Does this line just verify that the code does not crash?

Reply:
Yes, I thought it is not necessary to test the result as well (since naming is already checked in the previous lines).

Is it sufficient? Do you think we should test it in a different way?